src/trivy.ts

```diff
@@ -0,0 +1,556 @@
+      subcommands: [
+        ...scanOptions,
+        severityOption,
+        inputOption,
+        ignoreUnfixedOption,
+        insecureOption,
+        removePkgsOption,
+        outputOption,
+        configPolicy,
+        ...skipOptions,
+        ...clientServerOptions,
+        remoteServerOption,
+      ],
+    },
+    {
+      name: ["server", "s"],
+      description: "Server mode",
+      subcommands: [
+        skipDBUpdateOption,
+        downloadDBOnlyOption,
+        resetOption,
+        cacheBackendOption,
+        ...tokenOptions,
+        listenOption,
+      ],
+    },
+    {
+      name: ["config", "conf"],
+      description: "Scan config files",
+      subcommands: [
+        templateOptions,
+        formatOptions,
+        severityOption,
+        outputOption,
+        exitCodeOption,
+        skipPolicyUpdateOption,
+        resetOption,
+        clearCacheOption,
+        ignoreFileOption,
+        timeoutOption,
+        ...skipOptions,
+        ...regoPolicyOptions,
+        {
+          name: "--file-patterns",
+          description: "Specify file patterns [$TRIVY_FILE_PATTERNS",
+          args: {
+            name: "filePatterns",
+          },
+        },
+        {
+          name: "--include-non-failures",
+          description:
+            "Enable more verbose trace output for custom queries [$TRIVY_TRACE]",
+        },
+        {
+          name: "--trace",
+          description:
+            "Enable more verbose trace output for custom queries [$TRIVY_TRACE]",
+        },
+      ],
+    },
+    {
+      name: ["plugin", "p"],
+      description: "Manage plugins",
+      subcommands: [
+        {
+          name: ["install", "i"],
+          description: "Install a plugin",
+          args: {
+            name: "plugin",
+            description: "URL | FILE_PATH",
+          },
+        },
+        {
+          name: ["uninstall", "u"],
+          description: "Uninstall plugin",
+          args: {
+            name: "plugin",
+          },
+        },
+        {
+          name: ["list", "l"],
+          description: "List installed plugin",
+        },
+        {
+          name: "info",
+          description: "Information about a plugin",
+          args: {
+            name: "plugin",
+          },
+        },
+        {
+          name: ["run", "r"],
+          description: "Run a plugin on the fly",
+          args: {
+            name: "plugin",
+          },
+        },
+        {
+          name: "update",
+          description: "Update an existing plugin",
+          args: {
+            name: "plugin",
+          },
+        },
+        {
+          name: ["help", "h"],
+          description: "Shows a list of commands or help for one command",
+        },
+      ],
+    },
+    {
+      name: "version",
+      description: "Print the version",
+    },
+    {
+      name: ["help", "h"],
+      description: "Shows a list of commands or help for one command",
```

Comment:
Probably his accepts an arg and I think it may be the `help` template, same for the other `help` subcommands
```suggestion
      description: "Shows a list of commands or help for one command",
      args: {
        template: "help",
        isOptional: true,
      },
```

Reply:
done :) thx